packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -445,7 +445,6 @@
   .mdc-text-field__input {
     padding: $padding-inset;
     padding-top: $padding-inset * 2;
-    border: 1px solid transparent;
```

Comment:
I'm not 100% sure, but in active state textarea should have `2px` border and if you just remove this border, the whole border will become `1px`

![screencast 2017-12-08 15-39-06](https://user-images.githubusercontent.com/1291032/33766330-12557fd4-dc2e-11e7-8919-8850488151cb.gif)